plugins/org.eclipse.elk.alg.radial/src/org/eclipse/elk/alg/radial/RadialLayoutProvider.java

```diff
@@ -53,12 +60,6 @@ public void layout(final ElkNode layoutGraph, final IElkProgressMonitor progress
         }
         layoutGraph.setProperty(RadialOptions.RADIUS, layoutRadius);
         
-        //add support for the NodeDimensionCaluclation
-        ElkGraphAdapter adapter = ElkGraphAdapters.adapt(layoutGraph);
-        layoutGraph.getWidth();
-        NodeDimensionCalculation.calculateLabelAndNodeSizes(adapter);   
-        NodeDimensionCalculation.calculateNodeMargins(adapter);
```

Comment:
The code replacing this changes the layouter's behavior in that it also sorts the port lists. Is that intended?

Reply:
Yes. I don't see any reason not to execute the port list sorting. Without giving it much thought though.